README.md

```diff
@@ -170,6 +172,32 @@ Can you think of more? Let's discuss in the issues section!
   ```
 </details>
 
+#### InstantExecutorExtension
+
+Support for `InstantTaskExecutorRule` for JUnit 5. 
+
+<details open>
+  <summary>Kotlin</summary>
+
+  ```kotlin
+  dependencies {
+      testImplementation("io.github.neboskreb:instant-task-executor-extension:1.0.0")
+  }
+  ```
+</details>
+
+<details>
+  <summary>Groovy</summary>
+
+  ```groovy
+  dependencies {
+      testImplementation 'io.github.neboskreb:instant-task-executor-extension:1.0.0'
+  }
+  ```
+</details>
+
+For more details see [instant-task-executor-extension](https://github.com/neboskreb/instant-task-executor-extension) at GitHub.
```

Comment:
```suggestion
For more details see [instant-task-executor-extension](https://github.com/neboskreb/instant-task-executor-extension) on GitHub.
```

Quick grammar preference! Also, as I mentioned in another ticket, this repo might be private, so please double-check its visibility

Reply:
Fixed